packages/components/style.css

```diff
@@ -130,14 +107,7 @@ html {
 
 /* fix for nextra navbar for HiveNavigation */
 :root {
-  --nextra-navbar-height: 90px;
-}
-
-@media (min-width: 768px) {
-  .nextra-sidebar-container,
-  .nextra-toc > .nextra-scrollbar {
-    top: var(--nextra-navbar-height);
-  }
```

Comment:
we don't need this because nextra already have these classes related on --nextra-navbar-height